database/data/dataGeneration/genLikesPOSTGRES.js

```diff
@@ -0,0 +1,50 @@
+const faker = require('faker');
```

Comment:
will you keep this code if you use cassandra?

Reply:
I was wondering if I should keep vs. delete it. It could be a helpful future reference but I could just check previous commits as reference.